packages/app-store/_utils/getDefaultLocations.test.ts

```diff
@@ -4,13 +4,12 @@ import { getGoogleMeetCredential, TestData } from "@calcom/web/test/utils/bookin
 
 import { describe, expect, it } from "vitest";
 
-import { DailyLocationType, MeetLocationType } from "@calcom/app-store/locations";
-
+import { DailyLocationType, MeetLocationType } from "../locations";
 import { getDefaultLocations } from "./getDefaultLocations";
 
 type User = {
   id: number;
-  email?: string;
```

Comment:
Shouldn't be optional since `SessionUser["email"]` was a string (not string | undefined)